### ISSUE TITLE
QA: Users not receiving verification codes in Malaysia 

### ISSUE THREAD

**Issue by nol166** [codercom/bugs#139]
**Describe the bug**
The user signed up with Google and entered in their phone number in Malaysia

**To Reproduce**
1. Go to coder.com
2. Sign up
3. Choose Google as the preferred login method
4. Enter Malaysian phone number

**Expected behavior**
A verification code is sent to the number

**Comment by ammario** [comment]
@coadler any insight?

**Comment by coadler** [comment]
It seems texts to Malaysia are returning `30008 Unknown error`. Twliio isn't offering too much insight into what could be causing this. I'm hoping the switch to Bandwidth will fix this

**Comment by levlaz** [comment]
This along with https://github.com/codercom/bugs/issues/205 seem like a great place for an operational feature flag. You could either notify users that signups are not currently supported for known "broken" countries. Or you could potentially go through a separate MFA flow for these particular users.

**Comment by coadler** [comment]
Yeah, I'm currently using feature flags to select between Twilio and different regional carriers for better international deliverability. This has helped a ton for countries like India, but it's hard to find one for each of our problem countries.

The other issue is that a fair amount of texts being reported as not received are marked as delivered by Twilio, which makes the problem just that much more annoying. Having a feature flag to notify users from problem countries sounds like a great solution to our problem until we can solve the deliverability issues. I'll definitely look into implementing it.

Also, checked out your blog and saw you were from cincy too :smile:

**Comment by trinhpham** [comment]
The same issue from Vietnam.
